src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -105,16 +107,13 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }, []);
 
   useEffect(() => {
-    const { path, params } = props.match;
+    const { params } = props.match;
     const newChrLocationStr = getChrLocationStr(props.chrLocation);
-    const newUrl =
-      path
-        .replace(':species', params.species)
-        .replace(':objSymbol', params.objSymbol) +
-      '?region=' +
-      newChrLocationStr;
-
-    props.history.replace(newUrl);
+    const newUrl = `/app/browser/${params.species}/${
+      params.objSymbol
+    }?region=${newChrLocationStr}`;
+
+    replace(newUrl);
```

Comment:
I suspect `replace` is an action that has to be `dispatch`-ed. See example with `push` in connecter-react-router repo:

https://github.com/supasate/connected-react-router/blob/66215b121ebee82f36baec74a552cd3149c6e33e/FAQ.md#with-storedispatch

Reply:
Calling the replace function from connected-react-route seems to be dispatching the action we need. Would that cause any issues?

Reply:
Ok, I now see that in the latest commit you put `replace` in `mapDispatchToProps`, and are using `props.replace`, which means that the replace action is now properly dispatched. Good.